eth_typing/__init__.py

```diff
@@ -16,14 +27,3 @@
     Hash32,
     HexAddress,
 )
-
-from .encoding import (  # noqa: F401
```

Comment:
Not sure why CI was unhappy with this now but not in the past.

Reply:
Probably a newer isort version. Ran related issues recently as well.